agent/functional_tests/util/utils_windows.go

```diff
@@ -120,7 +120,7 @@ func (agent *TestAgent) StartAgent() error {
 			os.Setenv(k, v)
 		}
 	}
-	agentInvoke := exec.Command(".\\agent.exe")
+	agentInvoke := exec.Command(".\\agent-ecs-agent.exe")
```

Comment:
You will also need to change the build scripts to match this.

Reply:
NVM disregard this. Our functional test scripts already expect agent to be built.

Reply:
agent-ecs-agent.exe is wrong name, it's amazon-ecs-agent.exe